lib/caller.js

```diff
@@ -1,9 +1,19 @@
 'use strict'
 
+const fs = require('fs')
+
 function noOpPrepareStackTrace (_, stack) {
   return stack
 }
 
+function isFileName (input) {
+  return typeof input === 'string' && fs.existsSync(input)
+}
+
+function isOutsideNodeModules (file) {
```

Comment:
Little nit: I would join `isFileName` and `isOutsideNodeModules` as follow to avoid extra context switch:

```js
function isOutsideNodeModules (file) {
  return typeof input === 'string' && fs.existsSync(input) && file.indexOf('node_modules') === -1
}}
```

Reply:
done